setup.py

```diff
@@ -51,7 +51,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-adventure',
-    version='0.4.2',
+    version='0.4.3',
```

Comment:
Lets not update release tag since requirement is set to development branch in platform here https://github.com/edx-solutions/edx-platform/blob/47479aa06c52e9008672242e2e20ce37432d56dc/requirements/edx/custom.txt#L14

The release version and tag can be updated be release manager while releasing this to stage/prod.

Reply:
Fixed